src/main/java/junitparams/internal/parameters/ParametersFromCustomProvider.java

```diff
@@ -28,11 +31,28 @@ public boolean isApplicable() {
     }
 
     private ParametersProvider instantiate(Class<? extends ParametersProvider> providerClass) {
-        try {
-            return providerClass.newInstance();
-        } catch (Exception e) {
-            throw new RuntimeException("Your Provider class must have a public no-arg constructor!", e);
+        Constructor<?>[] constructors = providerClass.getConstructors();
+        for (Constructor<?> constructor : constructors) {
```

Comment:
yes .when there are two paremeters  ,there will be wrong.

But what is the second parameter?the orgin logic only suppert no-args constructor,and then support 1-args FrameworkMethod.if there will other requirement,we must add code